doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Ansible/Helm add api command  Proposal for Operator SDK
+authors:
+  - "@bharathi-tenneti"
+reviewers:
+  - "@fabianvf"
+  - "@cmacedo"
+approvers:
+  - "@fabianvf"
+  - "@jlanford"
+  - "@dmesser"
+creation-date: 2020-03-10
+last-updated: 2020-03-11
+status: implementable
+---
+
+
+# Ansible/Helm add api command  Proposal for Operator SDK
+
+
+## Release Signoff Checklist
+
+- \[x\] Enhancement is `implementable`
+- \[x\] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Accceptance criteria
+- \[ \] User-facing documentation is created
+
+
+
+## Summary
+
+In current scenario, users are not able to add additional APIs for Ansible/Helm based opertators using SDK CLI. This change will make it easier to add additional apis using CLI for Ansible and Helm based operators.
+
+## Motivation
+
+For Ansible/Helm operator SDK, user is not able to add additional apis for already scaffolded operators.
+
+## Goals
+
+The goal of the proposal if implemented, is to enable users add additional APIs once the original Ansible/Helm operator is scaffolded.
+
+## Non-Goals
+NA
+
+## Proposal
+
+### User Stories 
+
+#### Story 1 - Ansible Operator-sdk
+Enable Ansible operator-sdk to scaffold additional api, once the oroginal Ansible operator project has been created, using bwloq command.
+`operator-sdk add api --kind <kind> --api-version <group/version>`  
+
+##### Tasks included:
```

Comment:
The following are the list of the files that need to be scaffold. It is not part of the Story. It is detailed information about how the implementation needs to be done.